internal/driver/glfw/window_desktop.go

```diff
@@ -297,6 +301,12 @@ func (w *window) getMonitorForWindow() *glfw.Monitor {
 }
 
 func (w *window) detectScale() float32 {
+	// check if DPI detection is disabled
+	env := os.Getenv(disableDPIDetectionEnvKey)
+	if len(env) > 0 && (env[0] == 'T' || env[0] == 't' || env[0] == '1') {
```

Comment:
So "Talse" or "tomatoes" is to be treated as true?

Reply:
I can switch to a full proper list of supported terms (1, true, TRUE, True, t, T)